pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxSendOperator.java

```diff
@@ -71,6 +71,15 @@ public class MailboxSendOperator extends BaseOperator<TransferableBlock> {
   private final DataSchema _dataSchema;
   private BaseOperator<TransferableBlock> _dataTableBlockBaseOperator;
 
+  // TODO: Deduct this value via grpc config and calculate this dynamically based on distribution method etc.
+  // Max block size in bytes to send content over mail box service.
+  // Set to 4M for now.
+  private static int _maxBlockSize = 4 * 1024 * 1024;
+
+  public static void testOnlySetMaxBlockSize(int maxBlockSize) {
```

Comment:
```suggestion
  public static void testOnlySetMaxBlockSizeMB(int maxBlockSizeMB) {
```
it's a bit confusing whether this is rows or size, but looking at the code looks like it's size